tests/kafkatest/services/kafka/kafka.py

```diff
@@ -994,7 +1000,7 @@ def thread_dump(self, node):
     def clean_node(self, node):
         JmxMixin.clean_node(self, node)
         self.security_config.clean_node(node)
-        node.account.kill_java_processes(self.java_class_name(),
```

Comment:
`kill_java_processes` command won't work if Kafka is running in the native mode.
Replacing it with `kill_process`.